packages/editor/src/client/atoms/colorPicker.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import nanoid from 'nanoid'
+
+interface ColorPickerProps {
+  withColor: (color: string) => void
```

Comment:
Do you mean by `withColor` the new color that you pick? The naming is not very clear to me

Reply:
@0815pascal `withXYZ` is quite a common naming in react. But @faebebin don't you think `setColor` would be more clear here?

Reply:
IMO withColor implies a callback getting passed the color as arg. Setcolor / Setemoji would rather mean passing a value as Prop to the component?